lib/scss_lint/linter/groupable_selector.rb

```diff
@@ -0,0 +1,64 @@
+# add missing comparison for literals
+module Sass::Script::Tree
+  class Literal
+
+    def == other
+      self.class == other.class and self.value == other.value
+    end
+  end
+end
```

Comment:
We should not be defining this override in the `GroupableSelector` linter. It's unorthodox and could change the behavior of other linters which rely on the current behavior of the `==` method (there may not be any right now, but that could change).

Can we have a helper method `same_literal?` defined on `GroupableSelector` that accomplishes the same effect?